src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -194,6 +194,7 @@ class RandomBlockOolWriter : public ExtentOolWriter {
     paddr_t offset;
     ceph::bufferptr bp;
     RandomBlockManager* rbm;
+    extent_len_t len;
```

Comment:
Looks to be an unnecessary field, length of bp is already available.

Reply:
Removed